plugins/woocommerce/src/Admin/API/Reports/Variations/Controller.php

```
@@ -404,6 +414,22 @@ public function prepare_item_for_export( $item ) {
+		 * Developers can use this filter to add custom columns to the exported CSV
+		 * and data grid for product variations.
+		 *
+		 * @since x.x.x
+		 *
+		 * @param array $export_columns Export columns as an associative array (column_id => label).
+		 *
+		 * @return array Modified export columns.
+		 */
+		return apply_filters(
+			'woocommerce_report_variations_prepare_export_item',
+			$export_columns
```

PHPMD (2.15.0)

Avoid unused local variables such as '$export_columns'. (Unused Code Rules)

(UnusedLocalVariable)